### PR TITLE
feat: add mysql resource

### PR DESCRIPTION
**Pull request by mateuszjenek**

### CODE REVIEW

File: examples/mysql/README.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+<!-- BEGIN_TF_DOCS -->
```
Comment (johanneswuerbach) [inline examples/mysql/README.md:1]: Missed this in the other PR, but could you add a small introduction like https://github.com/humanitec-architecture/resource-packs-aws/tree/main/examples/mysql/aurora#example-mysql-resource-based-on-aws-rds-aurora?

**Review by johanneswuerbach — APPROVED** [review]
Two small remarks, otherwise 👍

File: modules/mysql/basic/provider.tf [hunk 2]
```diff
@@ -0,0 +1,19 @@
+terraform {
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "3.91.0"
```
[the remark this answers is not in the document]
Comment (mateuszjenek) [reply modules/mysql/basic/provider.tf]: Done!

File: examples/mysql/README.md (hunk 1 above)
Comment (mateuszjenek) [reply examples/mysql/README.md:15]: Added!

**Pull request merged**